src/ansys/geometry/core/_grpc/_services/base/repair_tools.py

```diff
@@ -42,6 +42,7 @@ class GRPCRepairToolsService(ABC):
 
     def __init__(self, channel: grpc.Channel):
         """Initialize the gRPC repair tools service."""
+        pass  # pragma: no cover
```

Comment:
@RobPasMue one your CI setups Codacy was giving warning for these lines for me so I haven't added it. FYI

Reply:
![image](https://github.com/user-attachments/assets/f497539f-7009-4325-8955-d2490c3ad9a5) for example, here it was complaining on these pass, but seems to be ok with the rest of it.

Reply:
good that it is not required because as seen it above from the image, results are apparently not always consistent.